020-LinuxFundamentals/Student/Challenge-12.md

```diff
@@ -0,0 +1,40 @@
+# Challenge 12 - Setting up a webserver 
+
+[< Previous Challenge](./Challenge-11.md) - **[Home](../README.md)** 
+
+## Description
+
+In this challenge we will be setting up a webserver and deploy a simple PHP application into it. As a plus, you can add SSL to ensure security requirements.
+
+- Download the sample application [from here](./resources/simple-php-app.tar.gz) to your home directory
```

Comment:
This is a direct link to the WTH repo!  The student guide should NOT have any links to the WTH repo, or links that will open GitHub in a browser.

Normally, the contents of /Student/Resources should be packaged into a "Resources.zip" file by the coach before the event.  In this case, since you only have one zip file, I will approve a direct link, but it must be a raw link.  You can do this by appending "?raw=true" to the end of the URL.